lib/v2/screens/wallet/components/tokens_cards/interactor/viewmodels/token_balances_bloc.dart

```diff
@@ -39,6 +43,9 @@ class TokenBalancesBloc extends Bloc<TokenBalancesEvent, TokenBalancesState> {
       yield TokenBalancesStateMapper().mapResultToState(state, potentialTokens, result);
     } else if (event is OnSelectedTokenChanged) {
       yield state.copyWith(selectedIndex: event.index);
+    } else if (event is OnFiatCurrencyChanged) {
+      yield state.copyWith(pageState: PageState.loading);
```

Comment:
This looks weird. 
SHow loading and instant show success?

Reply:
I saw that elsewhere - to trigger a reload. 

Not sure what the best way to do it is?!